gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/ApplicationServiceImpl.java

```diff
@@ -863,14 +863,18 @@ private Set<ApplicationEntity> convert(ExecutionContext executionContext, Collec
         }
 
         //find primary owners usernames of each applications
-        final List<String> appIds = applications.stream().map(Application::getId).collect(toList());
+        final List<String> appIds = applications
```

Comment:
use search to avoid filtering in stream ?

Reply:
sorry we can't